### ISSUE TITLE
How to update the certificates?

### ISSUE THREAD

**Issue by EugenMayer** [janeczku/rancher-letsencrypt#14]
How would you update the certificates retrieved, e.g. adding a subdomain? Would you create a new services for that, or remove the old stack and redo it with the addition domain?

**Comment by janeczku** [comment]
You would remove the stack, create a new letsencrypt service using the same name as "certificate name". This should update the previous certificate in Rancher's certificate store and upgrade any load balancers using the certificate. Let me know it that works for you!

**Comment by EugenMayer** [comment]
It does work perfectly. what i did was

1. exporting the configuration of the service using the gui ( as zip ) and extracting it.
2. deleting the old stack
3. create a new stack ( custom ) and add the rancher/docker yml file from the backup, adjusting the domains

this way its very efficient, not reentering api tokens and so on. 

And yes, that worked out, the certificate has been replaced (Infrastructure->Certificates) and yes, the loadbalancer has been automatically picking it up and the new domains are working out of the box.

It works, but i would not call it very convenient :)

I used a shell to check if there are any configuration files on the container, but the one i found under /etc/letsencrypt wont allow me adding anything, AFAICs
